src/Forms/UWP/ArcGISRuntime.Xamarin.Forms.UWP.csproj

```diff
@@ -12,8 +12,8 @@
     <AssemblyName>ArcGISRuntimeXamarin</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.10586.0</TargetPlatformVersion>
```

Comment:
This needed to change here (and in other places, too) because of a requirements change for the runtime.